development/stream_interface/multiplexer_demo.py

```diff
@@ -1,129 +1,121 @@
-                previous_frames[3],
-            ],
-            axis=1,
-        )
-        second_row = np.concatenate(
-            [
-                previous_frames[4],
-                np.zeros((480, 10, 3), dtype=np.uint8),
-                previous_frames[5],
-                np.zeros((480, 10, 3), dtype=np.uint8),
-                previous_frames[6],
-                np.zeros((480, 10, 3), dtype=np.uint8),
-                previous_frames[7],
-            ],
-            axis=1,
-        )
-        third_row = np.concatenate(
-            [
-                previous_frames[8],
-                np.zeros((480, 10, 3), dtype=np.uint8),
-                previous_frames[9],
-                np.zeros((480, 10, 3), dtype=np.uint8),
-                previous_frames[10],
-                np.zeros((480, 10, 3), dtype=np.uint8),
-                previous_frames[11],
-            ],
-            axis=1,
-        )
-        forth_row = np.concatenate(
-            [
-                previous_frames[12],
-                np.zeros((480, 10, 3), dtype=np.uint8),
-                previous_frames[13],
-                np.zeros((480, 10, 3), dtype=np.uint8),
-                previous_frames[14],
-                np.zeros((480, 10, 3), dtype=np.uint8),
-                previous_frames[15],
-            ],
-            axis=1,
-        )
-        image = np.concatenate(
-            [
-                first_row,
-                np.zeros((10, 2590, 3), dtype=np.uint8),
-                second_row,
-                np.zeros((10, 2590, 3), dtype=np.uint8),
-                third_row,
-                np.zeros((10, 2590, 3), dtype=np.uint8),
-                forth_row,
-            ],
-            axis=0,
+    for camera in cameras:
+        camera.start()
+    multiplexer = multiplex_videos(
+        videos=cameras,
+        batch_collection_timeout=None,
+        should_stop=lambda: STOP,
+    )
+    fps_monitor = sv.FPSMonitor(sample_size=8192)
+    monitors = {}
+    for i in range(n):
+        monitors[i] = sv.FPSMonitor(sample_size=8192)
+    for frames in multiplexer:
+        registered_frames = {}
+        for f in frames:
+            monitors[f.source_id].tick()
+            i = cv2.putText(
+                f.image,
+                f"LATENCY: {round((datetime.now() - f.frame_timestamp).total_seconds() * 1000, 2)} ms",
+                (10, f.image.shape[0] - 50),
+                cv2.FONT_HERSHEY_SIMPLEX,
+                2.0,
+                (0, 255, 0),
+                4,
+            )
+            i = cv2.putText(
+                i,
+                f"THROUGHPUT: {round(monitors[f.source_id](), 2)}",
+                (10, f.image.shape[0] - 120),
+                cv2.FONT_HERSHEY_SIMPLEX,
+                2.0,
+                (0, 255, 0),
+                4,
+            )
+            registered_frames[f.source_id] = i
+        for _ in range(len(frames)):
+            fps_monitor.tick()
+        fps_value = fps_monitor()
```

Comment:
I am seeing many errors in my console due to this method being deprecated. Should we replace all calls of FPSMonitor() with fps_monitor.fps? It will require all users to upgrade to the latest version of supervision, but without this change, it could confuse users to see this output when they aren't calling `FPSMonitor()` directly.

Reply:
we cannot change yet, solution for that is `SUPERVISON_DEPRECATION_WARNING=0` - we will work to make supervision less verbose. I do not want to bump sv straight ahead without learning about consequences for older environments

Reply:
I will evaluate implications and do it next release if possible. in Jetpack 4.6 we cannot install lib even now, as it supports py3.6, and supervision works with >= 3.8

Reply:
That makes sense. Deprecations can cause real headaches. Making supervision less verbose would be great.